interface/patient_file/summary/add_edit_issue.php

```diff
@@ -840,5 +841,22 @@ function divclick(cb, divid) {
  Calendar.setup({inputField:"form_end", ifFormat:"%Y-%m-%d", button:"img_end"});
  Calendar.setup({inputField:"form_return", ifFormat:"%Y-%m-%d", button:"img_return"});
 </script>
+
+<!--Page Form Validations-->
+<?php require_once($GLOBALS['srcdir']."/validation/validation_script.js.php");?>
```

Comment:
If we want to use the new validation (validate js) then we require this file to override the function validate in line 594(function validate())
If we move it up the function (validate in line 594) will be used ALWAYS as a validation mechanism .
We wanted to keep the old functionality like in Demographics and History.